### PR TITLE
Changed default installation path to /usr/local/bin.

### PR DESCRIPTION
**Pull request by vishwas-ftx**

### CODE REVIEW

File: maintainer.sh [hunk 1]
```diff
@@ -1,16 +1,25 @@
 #!/bin/sh
-set -x
-
 aclocal # Set up an m4 environment
 autoconf # Generate configure from configure.ac
 automake --add-missing # Generate Makefile.in from Makefile.am
 
 #Test configure & build
-./configure --prefix=$PWD CC=clang
+./configure CC=clang
```
Comment (kevinlahey) [inline maintainer.sh]: Any particular reason we're using clang?

File: maintainer.sh [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 make
-make install
+
+#install sgxtop / sgxstat link
+sudo make install
```
Comment (kevinlahey) [inline maintainer.sh]: I suspect that we should just skip the install altogether.  If someone wants to do a sudo, it should probably be explicit.

I think that most administrators would expect to do a separate "sudo make install" for the package.

**Review by kevinlahey — COMMENTED** [review]
Thanks for the fix!

File: maintainer.sh (hunk 2 above)
Comment (vishwas-ftx) [reply maintainer.sh]: done

File: maintainer.sh (hunk 1 above)
Comment (vishwas-ftx) [reply maintainer.sh]: Original makefile used clang. I've pushed a change removing that.

**Review by kevinlahey — COMMENTED** [review]
Can't believe I left this sitting here.  My apologies!

Could we please remove the sudos, even from commented out section.  I think that an unexpected sudo will startle users;  if you have sudo, you can type it yourself for "make install" and "make uninstall".

**Comment by vishwas-ftx** [comment]
> 

> Can't believe I left this sitting here. My apologies!
> 
> Could we please remove the sudos, even from commented out section. I think that an unexpected sudo will startle users; if you have sudo, you can type it yourself for "make install" and "make uninstall".

Done. No issues.

**Review by kevinlahey — APPROVED** [review]
Looks great, thanks!

**Pull request merged**